### DISCUSSION TITLE
I must be doing search wrong.

### DISCUSSION THREAD

**Discussion by mkrier** [brackets-cont/brackets#311] [General]
I want to do something very simple.  I want to search backwards from where the cursor is.  I press ctrl-f and type the term (I know for a fact there are 20+ instances of the term).  The first result is in the middle of the list.  I just want to search backwards from where I am at.  The left and right arrows next to the search box don't do anything when I click on them.
Should I convert this to a bug report?  
All I want to do is search backwards.  I have such a vanilla platform; I can't be the only one. 

Windows 11, 
Brackets Release 2.2 build 2.2.1-17943 (master c7a435e49)

**Comment by abose** [comment]
In all editors, ctrl+f to search, press enter ro move to next result, shift+enter moved back. The arrow keys typically is used to navigate within the search text box if searching in a single file in most editors.

**Reply by abose** [reply]
This would be a good feature to support navigation with up and down arrow keys too. I created an issue here https://github.com/phcode-dev/phoenix/issues/1291

**Comment by mkrier** [comment]
Thanks good idea.
I also noticed it doesn't have drag-and-drop for moving text like most 
other editors do.
Also it doesn't have an icon to click on to save like most editors do.
You might say well just press ctrl-s, but if I'm doing an edit that only 
involves the mouse (I do a *lot* of that daily), then it's tedious, (and 
unconventional) to not have one,

m

**Reply by abose** [reply]
Please create feature requests in this repo issues section - new issue

 https://github.com/phcode-dev/phoenix

Will check it out in the next releases.

**Reply by abose** [reply]
Fixed , will be available in feb release of pheonix desktop.

**Comment by charlypa** [comment]
@mkrier 
We've updated our editor to Phoenix Code, launched just yesterday. It now supports Windows, Mac, and Linux moving beyond the older Brackets x86 binaries.
 You can download Phoenix Code at https://phcode.io/.
